misk/src/main/kotlin/misk/web/jetty/ConnectionMetrics.kt

```diff
@@ -0,0 +1,53 @@
+package misk.web.jetty
+
+import com.google.common.annotations.VisibleForTesting
+import misk.metrics.Metrics
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+internal class ConnectionMetrics @Inject internal constructor(metrics: Metrics) {
+  val acceptedConnections = metrics.counter(
+      "jetty_connections_total",
+      "total number of connections accepted by jetty",
+      labels
+  )
+  val connectionDurations = metrics.summary(
```

Comment:
I think we want a `histogram`

Reply:
@alecholmes  we should actually just get rid of `summary` since we can't aggregate it

Reply:
And we'll get one...as soon as histograms are finished :)

Reply:
got it. you prefer the summary for now. makes sense